docs/admob/usage/index.md

```diff
@@ -20,7 +20,7 @@ yarn add @react-native-firebase/app
 yarn add @react-native-firebase/admob
 ```
 
-> On Android, you must update the "Contains ads" setting in the Google Play Store dashboard before releasing your app (under "Pricing & Distribution").
+> On Android, before releasing your app, you must select _Yes, my app contains ads_ radio in the Google Play Store, Policy, App content, Manage under Ads.
```

Comment:
```suggestion
> On Android, before releasing your app, you must select _Yes, my app contains ads_ in the Google Play Store, Policy, App content, Manage under Ads.
```

Just removing the implementation detail of how you specify it, but I like the update! I'll commit this and merge it thank you!